client/cd-it8.c

```diff
@@ -438,7 +439,7 @@ cd_util_create_sp (CdUtilPrivate *priv,
 	if (dot != NULL)
 		*dot = '\0';
 	cd_it8_set_title (cmf, title);
-	cd_it8_set_enable_created (cmf, FALSE);
+	cd_it8_set_enable_created (cmf, priv->timestamps);
```

Comment:
This is much better, thanks. Can you squash this patch down with the previous patch please.